L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrelatorLayer1Producer.cc

```diff
@@ -905,10 +964,14 @@ void L1TCorrelatorLayer1Producer::putEgObjects(edm::Event &iEvent,
       tkem.setPFIsol(egiso.floatRelIso(l1ct::EGIsoObjEmu::IsoType::PfIso));
       tkem.setPFIsolPV(egiso.floatRelIso(l1ct::EGIsoObjEmu::IsoType::PfIsoPV));
       tkems->push_back(tkem);
+      npho_obj.push_back(tkems->size() - 1);
     }
+    // FIXME: need realistic coordinates?
```

Comment:
could be helpful to just propagate the coordinates of one of the regions of the board, so that e.g. one can see from that if it's the positive or negative endcap, or which phi/eta slice of the barrel it is

Reply:
Yes, indeed, this is also coming. In fact, eta and phi are the only "characteristics" of the board propagated by the `RegionalOutput` container, hence needed to identify the board in the L2 producer.